lib/pxlogparser.js

```diff
@@ -0,0 +1,24 @@
+const PxConfig = require('./pxconfig');
+
+class PxLogsParser {
+    constructor(config) {
+        this.config = new PxConfig(config, null);
+
+    }
+    enrichLogs(logs) {
+        return logs.map((log) => this.enrichLogRecord(log));
+    }
+
+    enrichLogRecord(log) {
+        log.message = log.message.substring(0, this.config.px_external_logger_max_message_size || log.message.length);
+        Object.assign(log, {
+            messageTimestamp: new Date().toISOString(),
+            appID: this.config.config.PX_APP_ID,
+            container: 'enforcer',
+            configID: this.config.config.REMOTE_CONFIG_ID,
```

Comment:
This class is called PxLogsParser but it is coupled to Remote Config feature because defaulty adding RC fields, they should be moved to an if statement (shouldn't be added by default if they were not set in the config).

Reply:
I don't think we should change it, this is for the remote configuration feature only for now, and If App id / remote config id or version are not present than I want to see it in the logs

Reply:
maybe naming it PxRemoteConfigLogsParser ?